### PR TITLE
#839 - Creation of relations in PDF editor

### PR DESCRIPTION
**Pull request by wpmp**
**What's in the PR**
- add relation creation by drag and drop in the frontend
- move creation of spans into an if clause (mouesdown)

**How to test manually**
* Open document in PDF editor
* Create a relation annotation between two span annotation by drag and drop

**Automatic testing**
* [ ] PR includes unit tests

**Documentation**
* [ ] PR updates documentation

### CODE REVIEW

File: inception-pdf-editor/src/main/js/pdfanno/src/core/src/UI/span.js [hunk 1]
```diff
@@ -274,35 +274,35 @@ window.addEventListener('DOMContentLoaded', () => {
       if (spanAnnotation) {
         spanAnnotation.deselect()
       }
-    }
```
Comment (wpmp) [inline inception-pdf-editor/src/main/js/pdfanno/src/core/src/UI/span.js]: @reckart as I only moved this code into an if clause, is it ok to not annotate this change with `//BEGIN INCEpTION` etc.?

**Comment by wpmp** [comment]
@reckart I think this is ready for a review

**Comment by wpmp** [comment]
@reckart it seems PDFAnno does not render relations from one span to itself.
I will look into it, maybe I can change this.

**Comment by wpmp** [comment]
@reckart I managed to enable rendering of relations from a span to itself in PDFAnno.
Although it does not look very nice I think this is ok for now.
This PR now is ready for a review.

**Comment by reckart** [comment]
Look ok in Chrome. In Safari, I can create relations, but the arrows do not appear on screen.

**Comment by wpmp** [comment]
@reckart ah, yes, you opened an issue regarding this at the pdfanno repo, right?
may ask do you receive any errors in the browser console log?

**Comment by reckart** [comment]
I have now also opened an issue in our repo: https://github.com/inception-project/inception/issues/961

**Comment by reckart** [comment]
> @reckart ah, yes, you opened an issue regarding this at the pdfanno repo, right?
do you receive any errors in the browser console log?

I don't see anything suspicious in the browser console.

**Pull request merged**